templates/OperationRequestBodySnippet.stencil

```diff
@@ -0,0 +1,6 @@
+let requestBodyjsonData = try! JSONEncoder().encode( {{ op.request.objectName }})
```

Comment:
We shouldn't put a `!` into production code. If the larger method throw we can just let this throw. If not we can just wrap this in a `do-catch` and log the error.